lib/simplifyExpression/fractionsSearch/divideByGCD.js

```diff
@@ -50,6 +50,7 @@ function divideByGCD(fraction) {
     '/', [intermediateNumerator, intermediateDenominator]);
   const status = Node.Status.nodeChanged(
     ChangeTypes.FIND_GCD, fraction, intermediateFraction);
+  status.newNode = Node.Status.resetChangeGroups(status.newNode);
```

Comment:
Yes good call re: another substep!

Would you mind providing a tiny bit more guidance re: the 3-node change groups? I played around with this for awhile but got stuck. 😬  I think I can turn off the default change group by adding `false` as an argument to Node.Status.nodeChanged, so that's good. But how do I add a change group to one of the inner nodes? Is it the Node.Status.childChanged function?